pkg/services/ngalert/schedule.go

```diff
@@ -221,42 +232,46 @@ func (ng *AlertNG) alertingTicker(grafanaCtx context.Context) error {
 
 type alertDefinitionRegistry struct {
 	mu                  sync.Mutex
-	alertDefinitionInfo map[int64]alertDefinitionInfo
+	alertDefinitionInfo map[string]alertDefinitionInfo
```

Comment:
I think the key can be a struct type with a int64 (orgid) and a string (uid), then the string joining and conversion can go away I perhaps?

Reply:
(or could be a tuple array if string [2]string (but I think I like the struct more). Slices can not be map keys but arrays can.